frontend/src/pages/LoginPage.jsx

```
@@ -1,16 +1,22 @@
 import { useState } from 'react';
 import { useNavigate, Link } from 'react-router-dom';
+
+import api, { login as loginApi, getMe } from '../api';
+
 import API, { login as loginApi, getMe } from '../api';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'loginApi'. Consider to delete it or rename it.

'loginApi' is defined here:


(lint/suspicious/noRedeclare)